Zend/zend_compile.c

```diff
@@ -3438,6 +3457,7 @@ static void zend_compile_assign(znode *result, zend_ast *ast) /* {{{ */
 	if (is_this_fetch(var_ast)) {
 		zend_error_noreturn(E_COMPILE_ERROR, "Cannot re-assign $this");
 	}
+	// TODO: Mark assignment to http_response_header?
```

Comment:
Isn't this handled via `zend_try_compile_cv` already? Or what is this TODO about?

Reply:
I think I added it while I was trying to figure out how assignment worked, but yes this should be handled now